_dashboards/dashboard/index.md

```diff
@@ -0,0 +1,126 @@
+1. In the **Save visualization** window, add a title for the visualization. For example, the title for the bar chart panel is Order Date.
+1.  Select **Save and return**.  
+
+The bar chart visualization is now saved and you are taken back to the dashboard. You'll see two visualizations on the dashboard, as shown in the following image: 
+
+![Dashboard showing visualizations combined in a single view]({{site.url}}{{site.baseurl}}/images/dashboards/new-dashboard.png)
+
+## Adding subsequent panels
+
+Continuing with the dashboard you created in the preceding steps, you'll add a visualization to the dashboard.
+
+1. From the dashboard toolbar, choose **Add**.
+1. From the **Add panels** window, choose **[eCommerce] Sales by Category**.
+1. Select `x` to close the **Add panels** window. 
+
+You'll see an area chart visualization display on the dashboard, as shown in the following image: 
+
+![Adding another panel to the dashboard]({{site.url}}{{site.baseurl}}/images/dashboards/new-area-viz.png)
+
+## Saving dashboards
+
+When you've finalized your dashboard, save it. If you're saving a new dashboard:
+
+1. In the toolbar, choose **Save**.
+2. On the **Save dashboard** window, enter the **Title**. The **Description** is optional.
+3. To save the time filter to the dashboard, select **Store time with dashboard**.
+4. Choose **Save**.
+
+## Customizing the look of a panel
+
+To customize the panels, you'll need to be in edit mode:
+
+1. Select **Edit** at top right in the toolbar. 
+
+Showing a legend can give readers more information, while hiding a legend can give it a cleaner look. If you want to show or hide the panel legend:
+
+1. To show the legend, 
+2. To hide the legend, select the list icon in the panel's lower left corner.
+
+If you want to change the color of the panel legend:
+
+1. Select the gear icon on the area chart panel.
+2. From the **Options** window, select **Edit visualization**.
+3. From the visualization legend, select a category and then select a color from the flyout. The area chart updates with your change.
+4. Select **Save and return**. 
+
+If you want to show or hide panel title or customize it without applying the changes across the dashboard overall:
+
+1. Select the gear icon on the panel.
+2. From the **Options** window, select **Edit panel title**.
+3. From the **Customize panel**, add a title under **Panel title** or toggle the **Show panel title** to hide the title.
+4. Choose **Save**.
+
+## Arranging panels
+
+To organize panels, arrange them side-by-side, or resize them, you can use these options:
+
+- To move, select and hold the panel title and drag to the new location.
+- To resize, select the resize icon in the panel's lower right corner and drag to the new dimensions.
+- To view in fullscreen mode, select the gear icon (edit mode) or vertical ellipsis () in the top right of the panel and select **Maximize panel**. To minimize the fullscreen mode, select the vertical ellipsis and select **Minimize**.
```

Comment:
should the ellipsis be in the parenthesis?

Reply:
Added